runtime/sunvmi/sunvmi.c

```diff
@@ -89,11 +89,9 @@ latestUserDefinedLoaderIterator(J9VMThread * currentThread, J9StackWalkState * w
 	J9Class * currentClass = J9_CLASS_FROM_CP(walkState->constantPool);
 	J9ClassLoader * classLoader = currentClass->classLoader;
 
-	/* Ignore jdk/internal/loader/ClassLoaders$PlatformClassLoader (Java 9+).
-	 * In Java 9+, vm->extensionClassLoader is the PlatformClassLoader.
-	 */
+	/* Ignore the extension/platform classloader. In Java 9+, vm->extensionClassLoader is the PlatformClassLoader. */
 	BOOLEAN isPlatformClassLoader = FALSE;
-	if ((J2SE_VERSION(vm) >= J2SE_V11) && (classLoader == vm->extensionClassLoader)) {
+	if (classLoader == vm->extensionClassLoader) {
 		isPlatformClassLoader = TRUE;
```

Comment:
This might be clearer if we eliminate the local variable `isPlatformClassLoader` and replace the reference `!isPlatformClassLoader` with `(classLoader != vm->extensionClassLoader)`.

Reply:
Updated.